### PR TITLE
3 get api

### PR DESCRIPTION
**Pull request by iwebsteraudio**

### CODE REVIEW

File: __tests__/int.test.js [hunk 1]
```diff
@@ -34,3 +35,16 @@ describe("GET /api/topics", () => {
...
   });
 });
+describe("GET /api/", () => {
+    test("When requesting the API, simply responds with the details of the various endpoints", () => {
+        return request(app)
+        .get("/api/")
+        .expect(200)
+        .then(({ body }) => {
+            const apiData  = body;
+            console.log(apiData, endPoints)
```
Comment (nc-saima-khanom) [inline __tests__/int.test.js:45]: remove logs before pushing

File: __tests__/int.test.js [hunk 2]
```diff
@@ -34,3 +35,16 @@ describe("GET /api/topics", () => {
...
+
+            expect(apiData).toEqual( endPoints );
```
Comment (nc-saima-khanom) [inline __tests__/int.test.js:47]: nice!

File: __tests__/int.test.js [hunk 3]
```diff
@@ -34,3 +35,16 @@ describe("GET /api/topics", () => {
       });
   });
 });
+describe("GET /api/", () => {
+    test("When requesting the API, simply responds with the details of the various endpoints", () => {
+        return request(app)
+        .get("/api/")
+        .expect(200)
+        .then(({ body }) => {
+            const apiData  = body;
```
Comment (nc-saima-khanom) [inline __tests__/int.test.js:44]: do we need this line?

File: controllers/news-controllers.js [hunk 4]
```diff
@@ -8,9 +10,13 @@ const sendTopicData = (req, res, next) => {
     .catch(next);
 };
 
+const sendApiData = (req, res, next) => {
+  res.status(200).send( apiData );
```
Comment (nc-saima-khanom) [inline controllers/news-controllers.js:14]: We should send back data to the client on a meaningful key e.g, they should be able to access it on body.endpoints :)

File: __tests__/int.test.js (hunk 1 above)
Comment (iwebsteraudio) [reply __tests__/int.test.js:45]: Little FYI - just ticking all my notes off - trying to finally get NC News finished :) Might come up as loads of notifications.

Comment (nc-saima-khanom) [reply __tests__/int.test.js:45]: No worries! I have unsubscribed now :) Good luck!

**Pull request merged**